frank-flow/src/frontend/src/app/shared/components/file-tree/file-tree.component.ts

```diff
@@ -172,7 +172,6 @@ export class FileTreeComponent implements OnInit, OnDestroy {
       const item: File = JSON.parse(itemValue);
 
       if (this.filesAreEqual(this.currentFile, item)) {
-        this.currentFileService.resetCurrentDirectory()
```

Comment:
I can see the use now.

It is to manually reset a directory. You cant change current file, so if you have pressed a dir and then the current file, you cant remove it. It was usefull after all.